packages/earth-auth/src/signin/Signin.ts

```diff
@@ -100,6 +112,46 @@ const lock: Auth0LockStatic = new Auth0Lock(AUTH_CONFIG.clientID, AUTH_CONFIG.au
         };
       },
     },
+    {
+      type: 'select',
+      name: 'country',
+      placeholder: 'country or region',
+      options: function (cb) {
+        axios.get(`${AuthEnv.apiUrl}/profile/countries`).then((response) => {
```

Comment:
Maybe it's simpler to use the built-in `fetch` lib, and remove the extra dependency for axios.